sdk_container/src/third_party/coreos-overlay/coreos/config/env/sys-block/open-iscsi

```diff
@@ -0,0 +1,22 @@
+cros_post_src_install_mv_files() {
+	mv "${ED}"/var/lib/iscsi/ "${ED}"/usr/share/ || die
+	rmdir "${ED}"/var/lib || die
+
+	dosym ../flatcar/etc/iscsi/iscsid.conf /usr/share/iscsi/iscsid.conf
```

Comment:
`/usr/share/flatcar/etc` is the lowerdir for the `/etc/ overlay, so `/usr/share/iscsi/iscsid.conf` should just point at `/etc/iscsi/iscsid.conf`? or is it needed at all?

Reply:
The others like `/usr/share/ssh/sshd_config` all point to `../flatcar/etc`. I don't know why, as it should still work like you say, and you wouldn't need that empty file. I was just being consistent.

Reply:
It's needed for backward compatibility, if some user already has a symlink at `/etc/iscsi/iscsid.conf` that points to `/usr/share/iscsi/iscsid.conf`. So if `/usr/share/iscsi/iscsid.conf` points to `/etc/iscsi/iscsid.conf`, then we have symlinks pointing at each other. That's why we should point `/usr/share/iscsi/iscsid.conf` to `/usr/share/flatcar/etc/iscsi/iscsid.conf`.

Reply:
If someone has /etc from old old versions of Flatcar, then their `/etc/iscsi/iscsid.conf` will be a symlink, that overwrites whatever the lower layer in `/usr/share/flatcar/etc` has.